packages/opentelemetry-exporter-collector/src/platform/node/utilWithGrpc.ts

```diff
@@ -64,6 +71,37 @@ export function onInitWithGrpc(
     });
 }
 
+export function metricInitWithGrpc(collector: CollectorMetricExporter): void {
+  collector.grpcMetricsQueue = [];
+  collector.isShutDown = false;
+  const serverAddress = removeProtocol(collector.url);
+  const metricServiceProtoPath =
+    'opentelemetry/proto/collector/metrics/v1/metrics_service.proto';
+  const includeDirs = [path.resolve(__dirname, 'protos')];
+  protoLoader
+    .load(metricServiceProtoPath, {
+      keepCase: false,
+      longs: String,
+      enums: String,
+      defaults: true,
+      oneofs: true,
+      includeDirs,
+    })
+    .then(packageDefinition => {
+      const packageObject: any = grpc.loadPackageDefinition(packageDefinition);
+      collector.metricServiceClient = new packageObject.opentelemetry.proto.collector.metrics.v1.MetricsService(
+        serverAddress,
+        collector.credentials
+      );
+      if (collector.grpcMetricsQueue.length > 0) {
```

Comment:
Optional: 
```suggestion
      if (collector.grpcMetricsQueue.length) {
```

Reply:
I think `> 0` does a better job of conveying intention.